python/hail/matrixtable.py

```diff
@@ -507,10 +508,21 @@ def __getitem__(self, item):
             if row_key is not None and col_key is not None:
                 return self.index_entries(row_key, col_key)
             elif row_key is not None and col_key is None:
+                warnings.warn('The mt[<row keys>, :] syntax is deprecated, and will be removed before 0.2 release.\n'
+                              '  Use one of the following instead:\n'
+                              '    mt.rows()[<row keys>]\n'
```

Comment:
Do we use this syntax anywhere in the tests, examples, or tutorials? Might be worth removing them now.

Reply:
Can I do this in a followup PR? I want to keep everything as small as possible

Reply:
Sure.